yarn.lock

```diff
@@ -183,11 +183,11 @@ __metadata:
   linkType: hard
 
 "@babel/helper-annotate-as-pure@npm:^7.22.5":
-  version: 7.24.7
-  resolution: "@babel/helper-annotate-as-pure@npm:7.24.7"
+  version: 7.22.5
```

Comment:
Probably don't want to be downgrading the `@babel/` packages; guess it might have been caused by an inconsistent merge with `develop`?

Reply:
I noticed this too.  I tried checking out the yarn.lock from latest develop and re-running yarn, but it kept coming back this way.  Will give it another try.

Reply:
OK should make more sense now https://github.com/MetaMask/metamask-extension/pull/26433/commits/388e008f331b8fbea9954dab89c3661548aca261